src/Emitter.ts

```diff
@@ -19,91 +19,298 @@ export enum Events {
+   * 
+   * @example
+   * ```js
+   * const hfInstance = HyperFormula.buildFromSheets({
+   *  MySheet1: [ ['=SUM(MySheet2!A1:A2)'] ],
+   *  MySheet2: [ ['10'] ],
+   * });
    *
+   * // define a function to be called when the event occurs
+   * const handler = ( ) => { console.log('baz') }
+   *
+   * // subscribe to the 'sheetRenamed' event, pass the handler
+   * hfInstance.on('sheetRenamed', handler);
+   *
+   * // rename a sheet to trigger the 'sheetRenamed' event,
+   * // the console prints 'baz' each time a sheet is renamed
+   * hfInstance.renameSheet(0, 'MySheet0');
+   *
+   * // unsubscribe from the 'sheetRenamed' event
+   * hfInstance.off('sheetRenamed', handler);
+   *
+   * // rename a sheet
+   * // this time, the console doesn't print anything
+   * hfInstance.renameSheet(1, 'MySheet1');
+   * ```
+   * 
    * @category Sheet
    */
   sheetRenamed: (oldDisplayName: string, newDisplayName: string) => any,
 
   /**
-   * The event is triggered upon adding a named expression with specified values and location.
+   * Occurs when a named expression with specified values and location is added.
    * 
    * @event
    * 
    * @param {string} namedExpressionName the name of added expression
    * @param {ExportedChange[]} changes the values and location of applied changes
    *
+   * @example
+   * ```js
+   * const hfInstance = HyperFormula.buildFromArray([
+   * ['42'],
+   * ]);
+   *
+   * // define a function to be called when the event occurs
+   * const handler = ( ) => { console.log('baz') }
+   *
+   * // subscribe to the 'namedExpressionAdded' event, pass the handler
+   * hfInstance.on('namedExpressionAdded', handler);
+   *
+   * // add a named expression
+   * // the console prints 'baz' each time a named expression is added
+   * const changes = hfInstance.addNamedExpression('prettyName', '=Sheet1!$A$1+100', 0);
+   *
+   * // unsubscribe from the 'namedExpressionAdded' event
+   * hfInstance.off('namedExpressionAdded', handler);
+   *
+   * // add another named expression
+   * // this time, the console doesn't print anything
+   * const changes = hfInstance.addNamedExpression('uglyName', '=Sheet1!$A$1+100', 0);
+   * ```
+   * 
    * @category Named Expression
    */
   namedExpressionAdded: (namedExpressionName: string, changes: ExportedChange[]) => any,
 
   /**
-   * The event is triggered upon removing a named expression with specified values and from an indicated location.
+   * Occurs when a named expression with specified values is removed and from an indicated location.
    * 
    * @event
    * 
    * @param {string} namedExpressionName the name of removed expression
    * @param {ExportedChange[]} changes the values and location of applied changes
+   * 
+   * @example
+   * ```js
+   * const hfInstance = HyperFormula.buildFromArray([
+   * ['42'],
+   * ]);
+   *
+   * // define a function to be called when the event occurs
+   * const handler = ( ) => { console.log('baz') }
```

Comment:
```suggestion
   * const handler = ( namedExpressionName, changes ) => { console.log('baz') }
```
Maybe we should use those arguments to `console.log(`${namedExpressionName} was removed and triggered changes`, changes)` or sth like that?